ConsoleApp1/ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace TeleprompterConsole
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            RunTeleprompter().Wait();
+        }
+        private static async Task ShowTeleprompter(TelePrompterConfig config)
+        {
+            var words = ReadFrom("sampleQuotes.txt");
```

Comment:
You should either have a path here or read an environment variable. You can't guarantee the working directory is the same as the one the app runs

Reply:
Fair enough - and I did have to move the file as I'd put it in too high a directory to start with.  But just copying the tutorial